pilot/pkg/serviceregistry/kube/controller/pod.go

```diff
@@ -111,6 +112,19 @@ func GetPodConditionFromList(conditions []v1.PodCondition, conditionType v1.PodC
 	return -1, nil
 }
 
+func (pc *PodCache) labelFilter(old, cur interface{}) bool {
```

Comment:
Why is this a filter? It seems like it should be added to the queue and processed in onEvent?